### ISSUE TITLE
Can `archive/StorageBlobs` branch be deleted?

### ISSUE THREAD

**Issue by antkmsft** [Azure/azure-sdk-for-c#2020]
Updated 14 months ago by Rick.

https://github.com/Azure/azure-sdk-for-c/branches/all

**Comment by RickWinter** [comment]
This branch should remain until we GA the storage blob feature.

**Comment by antkmsft** [comment]
`archive/StorageBlobs` is not the `feature/storage-blobs`.

Archive is the old code that we pulled. It actually used to be named `feature/StorageBlobs`.
Feature is the "new" code that we created just recently - it pulled the code from the `feature/StorageBlobs` and incorporated one year of `main` changes there, but it has the new git history (using the old one without rebasing would complicate the history, plus it was PascalCased unlike other feature branches).

I renamed it so `archive/StorageBlobs` so that there is less confusion with the `feature/storage-blobs`. We used the code that was there to release the new code. I did not delete it, because there could've been people using it, so there is something for them to temporarily use if they don't want to make a switch right now.

Given all that, I think it is good to keep the `archive/StorageBlobs` branch around until we merge the `feature/storage-blobs` to `main`.
